modules/installation-vsphere-infrastructure.adoc

```diff
@@ -21,7 +21,7 @@ You must install the {product-title} cluster on a VMware vSphere version 6 or 7
 
 |Networking (NSX-T)
 |vSphere 6.5U3 or vSphere 6.7U2 and later
-|vSphere 6.5U3 or vSphere 6.7U2+ are required for {product-title}. VMware's NSX Container Plug-in (NCP) 3.0.2 is certified with {product-title} 4.6 and NSX-T 3.x+.
+|vSphere 6.5U3 or vSphere 6.7U2+ are required for {product-title}. VMware's NSX Container Plug-in (NCP) 3.0.2 is certified with {product-title} 4.7 and NSX-T 3.x+.
```

Comment:
Just wondering if we need to be concerned about something being certified with a particular OCP version, or if it applies okay going forward.

Reply:
This was updated just prior to the 4.6 release. Let me double check with the team to be sure.

Reply:
@ahardin-rh Just got word back from the team, they are advising we update this to keep 4.6 and remove the specific NCP version:
>vSphere 6.5U3 or vSphere 6.7U2+ are required for {product-title}. VMware’s NSX Container Plug-in (NCP) is certified with {product-title} 4.6 and NSX-T 3.x+.

Just let me know if you want to make that edit in this PR, or I can follow-up in a separate PR with that change.

Reply:
@codyhoag Excellent. Thank you! I can cover that update here. Thanks again!